src/lib389/lib389/cli_ctl/dblib.py

```diff
@@ -48,6 +49,27 @@ def __init__(self, *args, **kwargs):
         self.__dict__ = self
 
 
+def get_bdb_impl_status():
+    backldbm = 'libback-ldbm'
+    bundledbdb_plugin = 'libback-ldbm'
+    robdb_symbol = 'bdbro_getcb_vector'
+    libdb = 'libdb-'
+    plgstrs = check_plugin_strings(backldbm, [bundledbdb_plugin, robdb_symbol, libdb])
+    if plgstrs[bundledbdb_plugin] is True:
+        # bundled bdb build
+        if find_plugin_path(bundledbdb_plugin):
+            return BDB_IMPL_STATUS.BUNDLED
+        return BDB_IMPL_STATUS.NONE
+    if plgstrs[robdb_symbol] is True:
+        # read-only bdb build
+        return BDB_IMPL_STATUS.READ_ONLY
+    if plgstrs[libdb] is True:
+        # standard bdb package build
+        return BDB_IMPL_STATUS.STABDARD
```

Comment:
Probably a type (`STANDARD`?).
Also, it's not in `src/lib389/lib389/_constants.py`, I think.

Reply:
:+1: Good catch!  i fixed the typo and renamed RPM into STANDARD in the Enum definition.